src/Widgets/UserBox.vala

```diff
@@ -20,8 +20,7 @@
 public class Session.Widgets.Userbox : Gtk.ListBoxRow {
     private const int ICON_SIZE = 48;
 
-    public Act.User? user { get; construct; }
-    public bool is_guest { get; construct; default = false; }
```

Comment:
Looking at the other changes, the is_guest makes it really readable, it would be possible to also use it like this:
```suggestion
public bool is_guest {
    get {
        return user == null;
    }
}
```